packages/ra-ui-materialui/src/auth/Login.tsx

```diff
@@ -85,16 +79,12 @@ class Login extends Component {
     // the componentDidMount, but if the ref doesn't exist, it will try again
     // on the following componentDidUpdate. The try will be done only once.
     // @see https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element
-    updateBackgroundImage = (lastTry = false) => {
+    updateBackgroundImage = () => {
```

Comment:
why did you remove the lastTry? See the comment above the function.

Reply:
First, because this code does not work at all, read the original carefully and look where we were passing `lastTry`. Second because TypeScript agrees with me

Reply:
then you should remove the comment above the method, too